tests/trestle/core/commands/author/profile_test.py

```diff
@@ -808,3 +808,21 @@ def generate_assemble(md_path: pathlib.Path) -> MarkdownNode:
 
     tree = generate_assemble(ac1_path)
     assert not tree.get_node_for_key('## Control this_should_appear_in_parts')
+
+
+@pytest.mark.parametrize('show_values', [True, False])
+def test_profile_resolve(tmp_trestle_dir: pathlib.Path, show_values: bool, monkeypatch: MonkeyPatch) -> None:
+    """Test profile resolve to create resolved profile catalog."""
+    test_utils.setup_for_multi_profile(tmp_trestle_dir, False, False)
+    cat_name = 'resolved_catalog'
+    command_profile_resolve = f'trestle author profile-resolve -n main_profile -o {cat_name}'
```

Comment:
More test scenarios would be beneficial, like non-existing profile or invalid profile

Reply:
added two failure tests